src/core/Mixer.cpp

```diff
@@ -368,74 +348,122 @@ void Mixer::deleteChannel( int index )
 	Engine::audioEngine()->doneChangeInModel();
 }
 
+void Mixer::toggleAutoTrackLink(int index)
+{
+	m_mixerChannels[index]->m_autoTrackLinkModel.setValue(! m_mixerChannels[index]->m_autoTrackLinkModel.value());
+}
+
+IntModel * Mixer::getChannelModelByTrack(Track * track)
+{
+	if( track->type() == Track::InstrumentTrack )
```

Comment:
You can also use a switch-case.

Reply:
Yes - but normally I would not use a switch statement to differ between only two outcomes.